src/components/__test__/Profile.test.js

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable no-unused-vars */
```

Comment:
- [ ] It is always a good practice to make sure you fix all linter errors instead of disabling them, A linter is great for identifying errors when you use standard rules. Remember, a linter analyzes your code for stylistic and programming errors against the rules it knows. 💯 😉

Reply:
@BenMukebo I have removed all the linter disabling lines now.